### ISSUE TITLE
Add sha256sum to releases assets

### ISSUE THREAD

**Issue by jooola** [airsonic/airsonic#1073]
## Problem description

In addition to the gpg checking procedure, it would be nice to simply check the integrity or compare remote files using a checksum. For example, with ansible, if the local file (airsonic.war) fingerprint matches a remote fingerprint, the file won't be downloaded again.

They are plenty of uses for this little fingerprint.

**Comment by muff1nman** [comment]
Yeah we can do that. Ideally we should add a maven step to generate the checksums and make note to upload the file in the `contrib/release.md` doc.
